tools/workflow/docker-version-compare.sh

```diff
@@ -91,10 +91,17 @@ elif [ "${TARGET_MAJOR}" -eq "${LATEST_MAJOR}" ]; then
     fi
 fi
 
-[ ${MAJOR_UPDATE} == "true" ] || [ ${MINOR_UPDATE} == "true" ] || [ ${PATCH_UPDATE} == "true" ] || [ ${SAME_VERSION} == "true" ] && ANY_UPDATE=true || ANY_UPDATE=false
+# If any of the updates are true, then LATEST_OR_NEWER = true
+# LATEST_OR_NEWER would be false if the target version is less than the current latest version
+if [ ${MAJOR_UPDATE} == "true" ] || [ ${MINOR_UPDATE} == "true" ] || [ ${PATCH_UPDATE} == "true" ] || [ ${SAME_VERSION} == "true" ]; then 
+    LATEST_OR_NEWER=true
+else
+    LATEST_OR_NEWER=false
+fi
 
 echo "::set-output name=major-update::${MAJOR_UPDATE}"
 echo "::set-output name=minor-update::${MINOR_UPDATE}"
 echo "::set-output name=patch-update::${PATCH_UPDATE}"
-echo "::set-output name=any-update::${ANY_UPDATE}"
+echo "::set-output name=any-update::${LATEST_OR_NEWER}"
```

Comment:
```suggestion
echo "::set-output name=latest-or-newer::${LATEST_OR_NEWER}"
```

Reply:
Good catch, I still need to extra cautious with bash